ui/playground/app/components/Container.vue

```diff
@@ -15,7 +16,7 @@ const { schema } = defineProps({
   schema: { type: null, required: true },
   data: { type: null, required: true }
 });
-defineModel({ type: null, required: false });
+const model = defineModel({ type: null, required: false });
```

Comment:
this is a bit weird. `Container` component is not expected to have a `model`, because it's just a container.